src/PublicApiAnalyzers/Core/Analyzers/Microsoft.CodeAnalysis.PublicApiAnalyzers.csproj

```diff
@@ -8,6 +8,9 @@
     -->
     <PackageId>*$(MSBuildProjectFile)*</PackageId>
     <MicrosoftCodeAnalysisVersion>1.2.1</MicrosoftCodeAnalysisVersion>
+
+    <!-- Work around PerformanceSensitiveAttribute.cs cannot resolve ValueTask<TResult>. -->
+    <NoWarn>$(NoWarn);CS1574</NoWarn>
```

Comment:
📝 Submitted #4895 as a draft to remove all these as soon as this PR is published